sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/baseSender.ts

```diff
@@ -98,6 +98,9 @@ export abstract class BaseSender {
         // Failed -- persist failed data
         if (statusCode === 429 || statusCode === 439) {
           this.networkStatsbeatMetrics?.countThrottle(statusCode);
+          return {
+            code: ExportResultCode.SUCCESS,
+          };
         }
         if (result) {
           diag.info(result);
```

Comment:
I think we also concluded that we have to use itemsAccepted and not itemsReceived for partial success? I also prefer to explicitly check for status code 206 instead.

Reply:
Good catch!